tests/test_optimizely.py

```diff
@@ -125,16 +145,41 @@ class InvalidErrorHandler(object):
   def test_init__v1_datafile__logs_error(self):
     """ Test that v1 datafile logs error on init. """
 
-    self.config_dict['version'] = project_config.V1_CONFIG_VERSION
+    self.config_dict['version'] = enums.DatafileVersions.V1
```

Comment:
I think we can get rid of this test case as it tests the exact same thing as the one below